packages/amplify-headless-interface/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "amplify-headless-interface",
+  "version": "1.0.0",
+  "description": "interfaces for amplify headless mode payloads",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "tsc",
+    "test": "jest",
```

Comment:
Since the schemas are immutable and are part of the source code, it doesn't really make sense to be snapshot tested. They should be checked at PR time and then they are constant after that. The tests of the validation logic (and by extension the correctness of the schemas) should happen in the headless-input package because that's where the validation logic is

Reply:
Got it. So I think we'll have this snapshot testing for each of the versions in the util-headless package. For the snapshot testing in there , let's have tests per version of a category then?

Reply:
Yeah, there will be a test suite for each version of each schema (add storage, update storage, add api, update api, etc). Each time a new version of the spec is made, we'll also need to include tests that ensure the previous version can be upgraded correctly